js/ui/form/ui.form.layout_manager.js

```diff
@@ -371,6 +372,8 @@ var LayoutManager = Widget.inherit({
                         .addClass(item.cssClass)
                         .appendTo($itemElement);
 
+                $itemElement.toggleClass(SINGLE_COLUMN_ITEM_CONTENT, that.isLayoutChanged(this));
```

Comment:
Please renaming the "isLayoutChanged" method. By it name unclear what this method makes for this scenario.